README.rst

```diff
@@ -63,14 +63,69 @@ Publishing
     git tag 0.5.1
     git push origin 0.5.1
 
-4. Register the package::
+``.pypirc`` file
+-----------------
+
+To ensure publication and testing of publications are submitted to the correct
```

Comment:
What I'm attempting to say is the `.pypirc` is used to ensure that you can submit your package to the correct server where the cases are:
- I want to test publishing my package (i.e. I send it to a "dummy" server and then I can test that I can `pip install` etc). In this case I will use the `testpypi` server.
- I want to actually publish my package. In which case I will use the `pypi` server.

The unfortunate part is doing commands like: 
`python setup.py register -r testpypi`
`twine upload dist/* -r pypi`

with the `-r` argument only seem to work if you have a `.pypirc` file created.